docs/deployment/images.md

```diff
@@ -55,3 +55,31 @@ root@dokku:~# dokku tags:deploy node-js-app v0.9.0
        http://node-js-app.dokku.me
 
 ```
+
+## Add Image Directly from Docker Hub
+
+You can alternatively add image pulled from Docker Hub and deploy app from it by using tagging feature.
```

Comment:
I'm not a native english speaker but I think this reads better: "Alternatively you can pull an image from Docker Hub and deploy it using the tagging feature."